clusterloader2/testing/density/config.yaml

```diff
@@ -55,42 +67,60 @@ steps:
       labelSelector: kind = latencyPod
 - name: Creating saturation pods
 # Create latency pods
+- measurements:
+  - Identifier: WaitForRunningLatencyRCs
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      kind: ReplicationController
+      labelSelector: kind = latencyRC
+      operationTimeout: 10m
 - phases:
   - namespaceRange:
       min: 1
       max: {{$namespaces}}
-    replicasPerNamespace: {{DivideInt .Nodes $namespaces}}
+    replicasPerNamespace: {{$latencyReplicas}}
     tuningSet: Uniform5qps
     objectBundle:
     - basename: latency-pod-rc
       objectTemplatePath: rc.yaml
       templateFillMap:
         Replicas: 1
+        RCKind: latencyRC
         PodKind: latencyPod
 - measurements:
-  - Identifier: WaitForLatencyPodRunning
-    Method: WaitForRunningPods
-    Params:
-      desiredPodCount: {{.Nodes}}
-      labelSelector: kind = latencyPod
-      timeout: 10m
-- measurements:
-  - Identifier: PodStartupLatency
-    Method: PodStartupLatency
+  - Identifier: WaitForRunningLatencyRCs
+    Method: WaitForControlledPodsRunning
     Params:
       action: gather
 - name: Creating latency pods
-# Delete pods
+# Remove latency pods
 - phases:
   - namespaceRange:
       min: 1
       max: {{$namespaces}}
     replicasPerNamespace: 0
-    tuningSet: Uniform100qps
+    tuningSet: Uniform5qps
     objectBundle:
     - basename: latency-pod-rc
       objectTemplatePath: rc.yaml
+- measurements:
+  - Identifier: WaitForRunningLatencyRCs
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: gather
+- measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: gather
+- measurements:
+  - Identifier: WaitForRunningLatencyRCs
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: stop
```

Comment:
Do we need to stop measurements explicitly?
This is additional 5 lines of the code (2 times) that are boilerplate.

If we need, can we introduce an action "gather-and-stop" (or sth like that)?

Reply:
I think in this case should be ok to not stop the measurement, although I'm not a fan of this approach.

I would prefer to add `Finalize()` function to measurement interface. This function would be called for every measurement instance during cleanup phase. Wdyt?

Reply:
> I would prefer to add Finalize() function to measurement interface. This function would be called for every measurement instance during cleanup phase. Wd

This also sounds fine.